# PHAROS Autonomous Vehicle Sensor Config
PHAROS_AGV:
  SENSOR_CONFIG:
    ############ LIDAR #############
    LiDAR: {
      # Velodyne VLP-16 Hi-RES LiDAR // LEFT
      VLP16_LEFT: {
          xL: -0.440699,
          yL: 0.397052,
          zL: 1.90953,
          rollL: 1.69162,
          pitchL: 44.9466,
          yawL: 136.579,
        },
      # Velodyne VLP-16 Hi-RES  LiDAR // MIDDLE
      VLP16_TOP: {
          xT: 0.0,
          yT: 0.0,
          zT: 0.0,
          rollT: 0.0,
          pitchT: 0.0,
          yawT: 0.0,
        },

      # Velodyne VLP-16 Hi-RES  LiDAR // RIGHT
      VLP16_RIGHT: {
          xR: -0.449885,
          yR: -0.416713,
          zR: 1.91294,
          rollR: 179.496,
          pitchR: 135.409,
          yawR: 44.0134,
        },

      DESKEW_FLAG: 0,
      MULTI_LIDAR: 2
    }

    ############# IMU ##############
    IMU: {
        # EB-IMU V5 9-DoF
        xR: 0.0,
        yR: 0.0,
        zR: -0.2,
        rollR: 0.0,
        pitchR: 0.0,
        yawR: 0.0,
      }

    ############ VISION ############
    # VISION
    VISION: {
      # BlackFly-S FLIR CAMERA // LEFT
      FLIR_LEFT: {
          xR: 0.0,
          yR: 0.27,
          zR: 0.0,
          rollR: 0.0,
          pitchR: 0.0,
          yawR: 0.0,
        },
      FLIR_RIGHT: {
      # BlackFly-S FLIR CAMERA // RIGHT
          xR: 0.0,
          yR: -0.27,
          zR: 0.0,
          rollR: 0.0,
          pitchR: 0.0,
          yawR: 0.0,
        },
      Logi_BRIO_4K: {
      # Logitech BRIO 4K CAMERA // CENTER
          xR: 0.0,
          yR: -0.27,
          zR: 0.0,
          rollR: 0.0,
          pitchR: 0.0,
          yawR: 0.0,
          imgHeight: 1920,
          imgWidth : 1080,
          Fx : 1.144810406181205e+03,
          Fy : 1.144669963436012e+03,
          Cx : 9.662573529698977e+02,
          Cy : 5.430427209650604e+02,
        },
      }
